packages/api-server/src/decorator.ts

```diff
@@ -0,0 +1,32 @@
+import { asyncSleep } from "./util";
+import path from "path";
+import fs from "fs";
+
+const Profiler = require("v8-profiler-node8");
```

Comment:
Any problem with https://github.com/hyj1991/v8-profiler-next ?

Seems https://github.com/hyj1991/v8-profiler-node8/blob/master/v8-profiler.js#L1 only require `v8-profiler-next` and do nothing more ?

And `v8-profiler-next` has typescript support

Reply:
seems the author rewrite `v8-profiler-node8` with `v8-profiler-next`, wll replace with `v8-profiler-next`

Reply:
updated